Description: Adds sed and awk binaries in the initramfs
 In Ubuntu, it seems that sed and awk are there by default, possibly in the
 built busybox. But in Debian, it's missing, and that is making growroot fail.
 This patch fixes it, until a better solution can be found.
 .
 Also fix path of udevadm.
Author: Thomas Goirand <zigo@debian.org>
Bug-Debian: https://bugs.debian.org/852569
Forwarded: not-needed
Date: 2017-09-28

--- a/growroot/hooks/growroot
+++ b/growroot/hooks/growroot
@@ -12,5 +12,9 @@
 copy_exec /sbin/sfdisk /sbin
 copy_exec /usr/bin/growpart /sbin
-copy_exec /sbin/udevadm /sbin
+copy_exec /bin/udevadm /sbin
+copy_exec /usr/bin/awk /bin
+copy_exec /bin/sed /bin
+copy_exec /bin/rm /bin
+copy_exec /bin/readlink /bin
 
 # vi: ts=4 noexpandtab
